src/primitives/redpallas.rs

```diff
@@ -101,7 +101,7 @@ impl<T: SigType> Eq for VerificationKey<T> {}
 
 impl<T: SigType> PartialOrd for VerificationKey<T> {
     fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
-        <[u8; 32]>::from(self).partial_cmp(&<[u8; 32]>::from(other))
+        Some(self.cmp(other))
```

Comment:
Why it is needed?

Reply:
`cargo clippy` showed an error message about that: `incorrect_partial_ord_impl_on_ord_type`. It suggests that the `PartialOrd` and `Ord` implementations should be consistent, the more detailed description of this error can be found here: https://rust-lang.github.io/rust-clippy/master/index.html#/incorrect_partial_ord_impl_on_ord_type